app/models/dialog_field_tag_control.rb

```diff
@@ -37,6 +37,10 @@ def force_multi_value
     !single_value?
   end
 
+  def multiselect?
+    force_multi_value
+  end
```

Comment:
Should we add this method or just use `force_multi_value` on line 71? Or, if we do want `multiselect?`, should we just change it to an alias?

Just seems like we're making a method for the name, since it does nothing other than call the other method that already exists.

Reply:
I'm adding it for consistency with dropdowns, which are already using the method.

But, I can change both :).

But, then it can't be `single_value?` which doesn't exist for dropdowns, and to me `force_multi` sounds misleading, we don't care about "forced" but about what we're actually using, that's why I introduced `multiselect?` in the first place.

(And in fact, with tagging, `force_multi_value` now gives the wrong value when the category doesn't allow multiselect, but the editor didn't know that.)

Reply:
Ah ok that makes sense then, maybe we just do `alias_method :multiselect?, :force_multi_value` then?

Reply:
ah, next PR then :)